js/ui/overview.js

```diff
@@ -791,6 +791,10 @@ const Overview = new Lang.Class({
             this.hide();
         else
             this.show();
+    },
+
+    getActivePage: function() {
```

Comment:
Yeah, it is used by SideComponent. It looks like we forgot to port this code over when we ported over SideComponent.